gobblin-runtime/src/main/java/org/apache/gobblin/runtime/api/FlowSpec.java

```diff
@@ -423,6 +420,16 @@ public boolean isScheduled() {
     return getConfig().hasPath(ConfigurationKeys.JOB_SCHEDULE_KEY);
   }
 
+  /**
+   * Create a new FlowSpec object with the added property defined by path and value parameters
+   * @param path key for new property
+   * @param value
+   */
+  public FlowSpec addProperty(String path, String value) {
```

Comment:
lgtm , one last comment.
lets rename it to `createFlowSpecWithProperty` and move it to FlowSpec.Utils

Reply:
Done